src/components/admin/CenterContent.tsx

```diff
@@ -1,15 +1,19 @@
 import React from "react";
 import classNames from "classnames";
 
-import "./CenterContent.scss";
-
 export const CenterContent: React.FC<React.HTMLProps<HTMLDivElement>> = ({
   children,
   className,
   ...rest
 }) => {
   return (
-    <div className={classNames("CenterContent", className)} {...rest}>
+    <div
+      className={classNames(
```

Comment:
Not sure if this provides any benefit whatsoever, maybe we can just replace it with 

```tsx
className={`CenterContent ${className}`}
```

Reply:
I'd prefer using `classNames` because using template string may cause to an unexpected behavior in case `className` is not defined for example.
```ts
`${undefined}` === "undefined" // , not ""
```